### ISSUE TITLE
Error in documentation example for .sbgrc file

### ISSUE THREAD

**Issue by bjoernpollex** [sbg/sevenbridges-python#2]
In the Quickstart section of the documentation there is this example of an `.sbgrc` file:

```ini
[sbpla]
api-url = 'https://api.sbgenomics.com/v2'
auth-token = 700992f7b24a470bb0b028fe813b8100

[cgc]
api-url = 'https://api.sbgenomics.com/v2'
auth-token = 910975f5b24a470bb0b028fe813b8100

```

This example doesn't work (using a proper `auth-token`), because the URL should not be quoted. I'm using Python 3, apparently the issue does not come up in Python 2.

**Comment by SenadI** [comment]
Will be fixed soon. Thank you for the report.

**Comment by SenadI** [comment]
Hi @bjoernpollex 
We fixed this in 0.2.0 release. You already communicated to me in another issue that you updated to that version so now there should be no problems.

Sorry for closing the ticket without communicating to you.
